lib/iris/coords.py

```diff
@@ -1866,12 +1866,7 @@ def data(self, data):
             raise ValueError('The data payload of a CellMeasure may not be '
                              'None; it must be a numpy array or equivalent.')
         if is_lazy_data(data) and data.dtype.kind in 'biu':
-            # Disallow lazy integral data, as it will cause problems with dask
-            # if it turns out to contain any masked points.
             # Non-floating cell measures are not valid up to CF v1.7 anyway,
```

Comment:
@lbdreyer Minor, but you can remove the ` anyway,` from the end of this comment.